BrickKit/bricks/src/androidTest/java/com/wayfair/brickkit/brick/DataModelTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.wayfair.brickkit.brick
+
+import org.junit.Assert.assertTrue
+import org.junit.Assert.fail
+import org.junit.Test
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+class DataModelTest {
+
+    private val dataModel: DataModel = object : DataModel() { }
+
+    @Test
+    fun testIsReady() {
+        assertTrue(dataModel.isReady())
+    }
+
+    @Test
+    fun testNotifyChange() {
+        val countDownLatch = CountDownLatch(2)
+        val listener1 = object : DataModel.DataModelUpdateListener {
+            override fun notifyChange() {
+                countDownLatch.countDown()
+            }
+        }
+
+        val listener2 = object : DataModel.DataModelUpdateListener {
+            override fun notifyChange() {
+                countDownLatch.countDown()
+            }
+        }
+
+        val removedListener = object : DataModel.DataModelUpdateListener {
+            override fun notifyChange() {
+                fail()
+            }
+        }
+
+        dataModel.addUpdateListener(listener1)
```

Comment:
can do 
with(dataModel) {
...
}

if you want to get rid of some of the boilerplate